<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorClass</class>
 <widget class="QMainWindow" name="EditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="qviewport" name="wg_viewport" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>1280</width>
      <height>720</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 255, 255);</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>720</y>
      <width>1280</width>
      <height>180</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>1280</width>
      <height>180</height>
     </size>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::South</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_log">
     <attribute name="title">
      <string>Log</string>
     </attribute>
     <widget class="QTextBrowser" name="tb_log">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>1280</width>
        <height>130</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>1200</width>
        <height>120</height>
       </size>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_log_level">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_log_group">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>0</y>
        <width>120</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_browser">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>1280</width>
       <height>100</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 200, 133);</string>
     </property>
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>qviewport</class>
   <extends>QWidget</extends>
   <header location="global">qviewport.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="editor.qrc"/>
 </resources>
 <connections/>
</ui>
